package.json

```diff
@@ -147,13 +147,13 @@
     "lint:fix": "yarn eslint --fix",
     "format": "yarn prettier --check .",
     "format:fix": "yarn prettier --write .",
-    "test:unit": "vitest run src",
-    "test:unit:watch": "vitest run src",
-    "test:e2e": "vitest run tests/e2e",
-    "test:e2e:watch": "vitest watch tests/e2e",
+    "test:unit": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest run src",
+    "test:unit:watch": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest run src",
+    "test:e2e": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest run tests/e2e",
+    "test:e2e:watch": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest watch tests/e2e",
     "test:all": "yarn test:unit && yarn test:e2e && yarn test:examples",
-    "test:examples": "vitest --config ./examples/vitest.examples.config.ts run tests/examples",
-    "test:examples:watch": "vitest --config ./examples/vitest.examples.config.ts watch tests/examples",
+    "test:examples": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest --config ./examples/vitest.examples.config.ts run tests/examples",
+    "test:examples:watch": "BEE_FRAMEWORK_INSTRUMENTATION_METRICS_DISABLED=true vitest --config ./examples/vitest.examples.config.ts watch tests/examples",
```

Comment:
I would add such variable to `.env.test` (just create it, it is going to be automatically used: see `tests/setup.ts`)